guidelines/dependency-management-guidelines.md

```diff
@@ -0,0 +1,57 @@
+## Runtime dependencies
+
+### 3rd party
+
+#### Package
+
+We want to compile against the lowest compatible version to catch breaking changes.
+
+PackageReference: `[Major.Minor.0, Major+1)`
+
+#### Tests
+
+We want to compile against the latest version to catch breaking changes and bugs. This is the likely version that our users would be on.
+
+PackageReference: `[Major.*, Major+1)`
+
+### Core
```

Comment:
Can we make the more clear? Perhaps "Downstreams referencing NServiceBus.Core" ?